authorizations/authorization-ranger/src/main/java/org/apache/gravitino/authorization/ranger/RangerAuthorizationPlugin.java

```diff
@@ -735,18 +735,29 @@ public Boolean onUserAdded(User user) throws AuthorizationPluginException {
   @Override
   public Boolean onUserRemoved(User user) throws AuthorizationPluginException {
     VXUserList list = rangerClient.searchUser(ImmutableMap.of("name", user.name()));
-    if (list.getListSize() == 0) {
-      LOG.warn("The user({}) doesn't exist in the Ranger!", user);
+    boolean isExist = false;
+    long userId = 0;
```

Comment:
I think using two variables to manage `userId` is difficult.
Maybe we can use a LONG type of user ID?

Reply:
OK.